js/tji-donate.js

```diff
@@ -20,10 +20,22 @@ jQuery(document).ready(function($){
 	});
 
 	// Verify form details upon submit
-	$('.tji-donation-submit').click(function() {
-		event.preventDefault();
+	$('.tji-donation-submit').click(function(e) {
```

Comment:
@LayaTaal so if you want to use built on validation you can remove your manual checking below and use something like 
```
$('#js-donationform').on('submit', function(e){
   e.preventDefault();
  //submission code here
});
```
(I prefixed the id with js for convention)

and for the html
```
<input type="text" name="first_name" id="first_name" required minlength="3" maxlength="25">
<input type="email" name="email" id="email" required>
```
For donation amounts, you'll have to use vanilla js to check for a numeric value > 0?

and for the css
```
.input.required:invalid {
  border:1px solid $tji-red-or-whatever-var-we-have-for-red;
}
```
or similar